PluginsAndFeatures/azure-toolkit-for-intellij/src/main/resources/com/microsoft/azure/toolkit/operation/title/mysql.properties

```diff
@@ -1,12 +1,8 @@
-mysql.create=create Azure Database for MySQL({0}) in subscription({1})
-mysql|database.list.server=list all Azure MySQL database schemas in server({0})
-mysql|region.list.supported=list Azure MySQL-supported regions
-mysql|version.list.supported=list Azure MySQL-supported versions
+mysql.server.create.task=create Azure Database for MySQL({0}) in subscription({1})
+mysql.database.list.server=list all Azure MySQL database schemas in server({0})
```

Comment:
no need for this line, since mysql|database is the right service name.

Reply:
mysql|database.list.server

Reply:
updated